gwave_math.hlsl

```diff
@@ -0,0 +1,132 @@
+#ifndef GWAVEHLSLINCLUDE_INCLUDED
+#define GWAVEHLSLINCLUDE_INCLUDED
+
+
+
+float CalculateWaveNumber(float wavelength)
+{
+	float g = 9.81;
+	float pi = 3.14159265358979323846;
+	return 2 * pi / wavelength;
+}
+
+float CalculateFrequency(float wavelength, float steepness)
+{
+	float g = 9.81;
```

Comment:
Suggest to make gravity precise as well (to be concise with PI):

9.80665

Reply:
done